Makefile

```diff
@@ -99,10 +104,16 @@ doc:
 docker-qa: export TEST_OPTS := $(TEST_OPTS)
 docker-qa: export GCOVR_ISOLATED_TEST := zkQEVaBpXF1i
 
+ifeq ($(CC),all)
+docker-qa:
+	$(foreach cc,$(filter-out all,$(AVAILABLE_CC)),make $@ CC=$(cc); )\
```

Comment:
Should probably use the `$(MAKE)` variable when calling itself recursively?

At this point I'm also fairly sure that we're stretching Make beyond what it can reasonably do. We should probably consider porting parts of the Makefile to a Python script, but maybe keep Make as a convenient frontend. Thoughts on this?

Reply:
Of course is`$(MAKE)` correct.
I'll adapt the make to be able to use all for every target.
I think we should change parts of the make to use tox or nox.